### DISCUSSION TITLE
[Need more info] How to adjust parameters in k-fold cross validation by NNI?

### DISCUSSION THREAD

**Discussion by CXX1113** [microsoft/nni#3803] [Ideas]

**Comment by ultmaster** [comment]
Would you elaborate on your problem? What parameters on what k-fold cross validation?

**Comment by scarlett2018** [comment]
> Would you elaborate on your problem? What parameters on what k-fold cross validation?

@CXX1113 - can you make your question more clear by answering @ultmaster 's further clarifications? thanks.
